composer.json

```diff
@@ -27,7 +27,8 @@
     },
 
     "require-dev": {
-        "mink/driver-testsuite": "dev-master"
+        "mink/driver-testsuite": "dev-master",
+        "phpunit/phpunit": "^6.0"
```

Comment:
Please remove this, because Travis CI has PHPUnit already and with this change now all PHP version fail.

Reply:
Removed (I'm still learning how travis is configured).